xla/service/gpu/runtime/command_buffer_cmd.h

```diff
@@ -784,6 +784,24 @@ class AllGatherCmd : public CollectiveCmd {
   std::vector<NcclCollectiveThunk::Buffer> buffers_;
 };
 
+//===----------------------------------------------------------------------===//
+// CollectiveBroadcastCmd
+//===----------------------------------------------------------------------===//
+
+class CollectiveBroadcastCmd : public CollectiveCmd {
+ public:
+  CollectiveBroadcastCmd(NcclApi* nccl_api, NcclCollectiveConfig config,
+                         absl::Span<const NcclCollectiveThunk::Buffer> buffers);
+
+  absl::Status Record(const Thunk::ExecuteParams& params, StateManager& state,
+                      se::CommandBuffer* command_buffer) override;
```

Comment:
I see several build breakages, e.g. `xla/service/gpu:ir_emitter_unnested` all failing with 
`error: non-virtual member function marked 'override' hides virtual member function
se::CommandBuffer* command_buffer) override;`

Reply:
Can I get my context here? I'm not sure what to fix.

Reply:
Fixed.